src/scripts/utils/FontFace.js

```diff
@@ -50,5 +50,15 @@ export function createFontFacesArray(ayahs) {
     }
   });
 
+  fontFacesArray.push(
+    `@font-face {font-family: 'bismillah';
+    src: url('/fonts/compressed/eot/bismillah.eot?#iefix') format('embedded-opentype'),
+    url('/fonts/ttf/bismillah.ttf') format('truetype'),
+    url('/fonts/woff/bismillah.woff?-snx2rh') format('woff'),
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.